panther_analysis_tool/main.py

```diff
@@ -489,6 +574,50 @@ def delete_analysis(args: argparse.Namespace) -> Tuple[int, str]:
         return 1, ""
 
     logging.info("Detection(s) %s have been deleted.", " ".join(analysis_id_list))
+    return 0, ""
+
+
+def delete_router(args: argparse.Namespace) -> Tuple[int, str]:
+    # Routes all things delete to the functions they need to go to
+
+    # Get lists of analysis and queries from args
+    analysis_id_list = args.analysis_id
+    query_list = args.query_id
+
+    # If we didn't get a list of analysis, look up analysis associated with queries that were passed
+    if len(analysis_id_list) == 0 and len(query_list) > 0:
+        analysis_id_list = get_analysis_id_by_query(args, query_list)
```

Comment:
so its not supported to deleted queries + regular rules/policies in a single command?